modules/cache/utils/cache.js

```diff
@@ -96,8 +97,10 @@ cache.keepHot = keepHotFunction => {
     logger.info(`cache keepHot ${new Date().getTime() - start}ms`);
   });
 
-  keepHotLogic();
-  setInterval(keepHotLogic, cache.keepHotInterval);
+  queueLogic.subscribeToMessage(MESSAGE_QUEUES.SCHEDULED_EVENTS, MESSAGES.SCHEDULED_EVENTS.COMMANDS.CACHE_KEEPHOT, async message => {
```

Comment:
it feels weird that the util subscribes to the scheduler, should we move this to `CacheLogic`?

Reply:
fine for me, I can adjust it

Reply:
all keephot is now moved to cache logic